system/partials/_map.scss

```diff
@@ -0,0 +1,18 @@
+@mixin map {
+    .ds-map {
+        height: 500px;
```

Comment:
You have to set a height on your div because by default it has a height of zero.
Both WDIV and EE use `500px` for this:
https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/polling_stations/assets/custom_css/map.css
https://github.com/DemocracyClub/EveryElection/blob/master/every_election/assets/css/map.css
I'm not really sure what is going on with WCIVF. It doesn't seem to be setting any height on the map div. Tbh, it might be completely broken but I don't have a quick way to test it.
Anyway, point is: I suggest we standardise on a `500px` high map as a default default and let individual applications override it if they need to.

Reply:
Not tested, but I wonder if:

```css
width: 100%;
aspect-ratio: 1 / 1;
```

Might be a better approach? You might need to play with the ratio to work well for different screen sized, but ultimately this should be nicer than setting a pixel height.

Reply:
I reckon 1:1 is probably about right tbh, and more likely to be a good **default** than hard-coded. If we get it into a context and it looks silly we can always override it.

Reply:
https://github.com/DemocracyClub/design-system/pull/147/commits/cba7fabe14226ffb9c3e99b02842be8e2523e02f